### DISCUSSION TITLE
Can't log in

### DISCUSSION THREAD

**Discussion by GlowSquidYT** [osudroid/osu-droid#198] [General]
I'm using my osu droid account 
And I can't login for some reason even if I change the password on the website it still doesn't work. on old version it doesn't work . even on the latest one it doesn't. pls help

**Comment by reacto11mecha** [comment]
Did you join the international community discord server? There was a problem going on if you read the announcement channel that one of the osu!droid server maintainers accidentally deleted all player data. But fortunately, there was a backup.

My advice is you need to join the discord server for more info, cause im not the moderator

**Reply by GlowSquidYT** [reply]
I'm banned on the server cause I joined when I was 12

**Reply by GlowSquidYT** [reply]
I was gonna say just mute me but I was already banned

**Reply by GlowSquidYT** [reply]
Nvm it's working again
